src/SoundPlayer.js

```diff
@@ -1,13 +1,16 @@
 var Tone = require('tone');
 var log = require('./log');
 
-function SoundPlayer (outputNode) {
-    this.outputNode = outputNode;
+function SoundPlayer () {
```

Comment:
I know this is all WIP, but I would recommend adding JSDoc comments to your classes & functions as you go so that there's less doc burden at the end.